specs/adding_congressional_district.md

```diff
@@ -0,0 +1,43 @@
+# Adding the Congressional District ID
+
+## Problem Statement
+
+Determine the congressional district for each loan in our database,
+based on stg_analytics.sba_sfdo (borr_street, borr_city, borr_state,
+borr_zip).
```

Comment:
and potentially latitude/longitude once we get that in there right? This PR is I think 80% ready to go: https://github.com/sfbrigade/datasci-sba/pull/25